.github/workflows/deploy.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy Plugin
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to deploy (e.g., 1.0.0)'
+        required: true
+        type: string
+      deploy_wporg:
+        description: 'Deploy to WordPress.org'
+        required: false
+        default: true
+        type: boolean
+      deploy_github:
+        description: 'Deploy to GitHub'
+        required: false
+        default: true
+        type: boolean
+      simulate:
+        description: 'Dry run mode'
+        required: false
+        default: true
+        type: boolean
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the private action repository
+      - name: Checkout woo-product-deploy action
+        uses: actions/checkout@v4
+        with:
+          repository: woocommerce/woo-product-deploy
+          ref: trunk
+          path: ./.github/actions/woo-product-deploy
+          token: ${{ secrets.ORG_DEPLOY_SECRET }}  # Need PAT with access to the action repository
+
+      # Use the action we checked out
+      - name: Deploy Product
+        uses: ./.github/actions/woo-product-deploy
+        with:
+          repo_url: https://github.com/woocommerce/reddit-for-woocommerce
+          branch: ${{ github.ref_name }}
+          version: ${{ github.event.inputs.version }}
+          node_version: '20'
+          npm_version: '10'
+          simulate: ${{ github.event.inputs.simulate }}
+          github_release: ${{ github.event.inputs.deploy_github }}
+          wporg_release: ${{ github.event.inputs.deploy_wporg }}
+          wporg_username: ${{ secrets.WPORG_USERNAME }}
+          wporg_password: ${{ secrets.WPORG_PASSWORD }}
+          deploy_pat: ${{ secrets.ORG_DEPLOY_SECRET }}
+          partner_user: ${{ secrets.ORG_PARTNER_USER }}
+          partner_secret: ${{ secrets.ORG_PARTNER_DEPLOY_SECRET }}
```

Comment:
As this is for deploy to WP.org only, `wccom_product_id` might not needed, but just wanted to check if you have confirmed it. (I don't have access to deploy action repo. So, not able to verify)

Reply:
Good question. We confirmed that this wasn't needed for the Google for WooCommerce extension. My understanding is that it is only need when we are deploying directly to WCCOM. When deploying to .org this is not necessary because there is another integration that syncs this to the WC marketplace.